packages/common/src/database/TableDefinitions.ts

```diff
@@ -31,7 +53,9 @@ export class TableDefinitions {
         // prettier-ignore
         this.CONTAINMENTS_COLUMN_SET = new this.pgp.helpers.ColumnSet(
             [
-                "?containment",         // Don't update this column
+                "?containment_language",  // The classifier of the node
```

Comment:
Duplicate comments that seem to be inaccurate. Maybe have just one comment "The following three fields together form the meta-pointer for the classifier of the node"?

Reply:
Changed text, kept repetition.